requirements.txt

```diff
@@ -4,7 +4,7 @@ Flask-Admin>=1.0.7
 Flask-Bcrypt
 Flask-Bootstrap
 Flask-Cache
-Flask-Login>=0.1.1
+Flask-Login==0.2.11
```

Comment:
Yes, I tested the app, but did not thoroughly check the unit tests. If we use 0.3 or later of Flask Login we run into a breaking change: https://github.com/maxcountryman/flask-login/blob/master/CHANGES#L25-L26